libs/common/src/sql-db-entities/fee-record-correction/fee-record-correction.entity.ts

```diff
@@ -113,6 +127,8 @@ export class FeeRecordCorrectionEntity extends AuditableBaseEntity {
     recordCorrection.additionalInfo = additionalInfo;
     recordCorrection.isCompleted = false;
     recordCorrection.updateLastUpdatedBy(requestSource);
+    recordCorrection.bankTeamName = bankTeamName;
+    recordCorrection.bankTeamEmails = bankTeamEmails;
```

Comment:
Could we pass the emails down as an array and then join them here? Feels a bit nicer to join them at the last moment.

Reply:
done